uktrade/data-hub-frontend
# Add pending D&B investigation message

## Description of change
When the `company.pending_dnb_investigation` is `true` then a status message needs to be displayed so that the user knows the company is pending investigation.

## Test instructions
When using Sandbox as the API:
- Browse to `~/companies/ca8fae21-2895-47cf-90ba-9273c94dab81/activity`, you will see the status message
 
## Screenshots
### Before
<img width="905" alt="Screenshot 2019-09-17 at 13 42 35" src="https://user-images.githubusercontent.com/1150417/65042640-2b16be80-d951-11e9-839a-db2834bc105b.png">

### After 
<img width="954" alt="Screenshot 2019-09-17 at 13 39 26" src="https://user-images.githubusercontent.com/1150417/65042655-336ef980-d951-11e9-8113-c59c9a9e4aba.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
